### PR TITLE
style: Remove google fonts (addresses #5025).

### PR DESCRIPTION
**Pull request by mgrhm**

### CODE REVIEW

**Comment by iloveitaly** [comment]
I'm curious why we need to remove these google fonts. Security / load speed?

**Comment by mgrhm** [comment]
In their (original request)[https://github.com/monicahq/monica/issues/5025], @krswin said it's important that their Monica instance doesn't connect to the internet for any types of requests.  I'd assume that they're hosting it locally and don't want Monica making HTTP calls outside their local network.

From my perspective, I can't see a good reason why these are included.  They don't improve the site's function (they're barely used), they add to load time, and they're making requests to Google's servers.  I don't see a benefit to any of those -- and I'd say removing Google from an application I self-host is a plus.

In this pull, I'm removing Google Fonts from an error page.  The page works just fine, and will load faster, without that additional overhead.

Self-hosting webfonts is a bit fiddly to set up, but not that difficult really.  If it's something that Monica wants/needs, we should add the fonts and code so that they're hosted and served locally.  As it stands, though, Monica's font-stack is pretty thorough and doesn't rely on non-system fonts.

**Comment by iloveitaly** [comment]
Makes sense! Thanks for explaining. Definitely in favor of this!

File: resources/views/errors/layout.blade.php [hunk 1]
```diff
@@ -15,10 +13,10 @@
                 margin: 0;
                 padding: 0;
                 width: 100%;
-                color: #B0BEC5;
+                color: #999999;
```
Comment (asbiin) [inline resources/views/errors/layout.blade.php:16]: Why changing the color here?

File: resources/views/errors/layout.blade.php [hunk 2]
```diff
@@ -15,10 +13,10 @@
...
                 display: table;
                 font-weight: 100;
-                font-family: 'Lato';
+                font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
```
Comment (asbiin) [inline resources/views/errors/layout.blade.php:19]: I suggest adding 'Lato Light' at first so it's quite the same as before on systems where the font exist.
```suggestion
                font-family: "Lato Light", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
```

**Review by asbiin — CHANGES_REQUESTED** [review]
Good work, thank you @mgrhm

File: resources/views/errors/layout.blade.php (hunk 1 above)
Comment (mgrhm) [reply resources/views/errors/layout.blade.php:16]: The colour #B0BEC5 is only used in that one instance – so I replaced it with its nearest neighbour as part of the standardisation process.

**Comment by asbiin** [comment]
Thank you @mgrhm

**Pull request merged**